code/game/objects/effects/decals/cleanable/misc.dm

```diff
@@ -445,7 +445,7 @@ GLOBAL_LIST_EMPTY(nebula_vomits)
 
 /obj/effect/decal/cleanable/ants/update_overlays()
 	. = ..()
-	. += emissive_appearance(icon, "[icon_state]_light", src, alpha = src.alpha)
+	. += emissive_appearance(icon, "[icon_state]_light", src, alpha = src.alpha, effect_type = EMISSIVE_NO_BLOOM)
```

Comment:
are you going to/have you updated all emissive_appearance calls to include this?

Reply:
Yeah I did a pass on all emissive_appearance calls